Native extension won’t compile on macOS 10.16/11.0 Big Sur beta

```
$ gem install rbtrace
...
rbtrace.c:1097:3: error: implicit declaration of function 'rb_postponed_job_register_one' is invalid in C99 [-Werror,-Wimplicit-function-declaration]
  rb_postponed_job_register_one(0, rbtrace__receive, 0);
  ^
2 warnings and 1 error generated.
make: *** [rbtrace.o] Error 1

make failed, exit code 2

Gem files will remain installed in /Users/george/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/gems/rbtrace-0.4.13 for inspection.
Results logged to /Users/george/.rbenv/versions/2.7.1/lib/ruby/gems/2.7.0/extensions/x86_64-darwin-20/2.7.0/rbtrace-0.4.13/gem_make.out
```

The proximate cause of this appears to be that `clang` from the Xcode 12 Beta Command Line Tools [now sets `-Werror=implicit-function-declaration` by default](https://developer.apple.com/documentation/xcode-release-notes/xcode-12-beta-release-notes#Resolved-Issues):

> Clang now reports an error when you use a function without an explicit declaration when building C or Objective-C code for macOS (`-Werror=implicit-function-declaration` flag is on). This additional error detection unifies Clang’s behavior for iOS/tvOS and macOS 64-bit targets for this diagnostic. (49917738)

Passing `-Wno-implicit-function-declaration` fixes:

```
$ gem install rbtrace -- --with-cflags=-Wno-implicit-function-declaration
Building native extensions with: '--with-cflags=-Wno-implicit-function-declaration'
This could take a while...
Successfully installed rbtrace-0.4.13
1 gem installed
```

…though it might be unintentional that we’re relying on the implicit declaration of `rb_postponed_job_register_one`.

--- a/ext/rbtrace.c
+++ b/ext/rbtrace.c
@@ -22,6 +22,7 @@
 
 #include <msgpack.h>
 #include <ruby.h>
+#include <ruby/debug.h>
 
 #ifndef RUBY_VM
 #include <env.h>

